bundles/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/internal/dataexchange/NukiApiServlet.java

```diff
@@ -114,15 +114,20 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
             responseEntity = new ResponseEntity(HttpStatus.BAD_REQUEST_400,
                     new NukiHttpServerStatusResponseDto("Invalid BCB-Request!"));
         } else {
-            responseEntity = doHandle(bridgeApiLockStateRequestDto, request.getParameter("bridgeId"));
+            try {
+                responseEntity = doHandle(bridgeApiLockStateRequestDto);
+            } catch (Exception e) {
```

Comment:
Probably more something only for you as developer to find a potential bug.

Reply:
I mean, yes, that's exactly why I added it. If someone reports problem with callback I won't be able to debug it if I don't know what was sent.

Reply:
But do you have an example of exception by a user? 
Why not rather analyzing the current code and only catch more specific exceptions? 
In case this change makes sense, logging at warn level would be more appropriate.

Reply:
As I've said I've added it because I found some Nullpointers in logfile which I can't replicate (which means it could happen to someone else) and knowing what data were sent would really help. I agree that warn would be better.

Reply:
Why not fixing the NPE in this case, rather than catching the  exception?

Reply:
I've fixed the one place it throws NPE but it still would be helpful to see the data sent since that boolean flag should not be null